Amplitude/Amplitude.m

```diff
@@ -257,6 +277,15 @@ - (void)initializeApiKey:(NSString*) apiKey
 
 - (void)initializeApiKey:(NSString*) apiKey userId:(NSString*) userId
 {
+    UIApplicationState state = [UIApplication sharedApplication].applicationState;
+    [self initializeApiKey:apiKey userId:userId startSession:(state != UIApplicationStateBackground)];
+}
```

Comment:
I think that's the preferred default behavior. We should bump the minor patch number (to 2.4) and make note of it in the release notes that session behavior has changed for push notifications. Are there other situations where the check would cause the startSession to not be logged?